kafka-cruise-control-stop.sh

```diff
@@ -3,7 +3,7 @@
 # See License in the project root for license information.
 
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep -i 'cruise-control' | grep java | grep -v grep | awk '{print $1}')
+PIDS=$(ps ax | grep -i 'KafkaCruiseControlMain' | grep java | grep -v grep | awk '{print $1}')
```

Comment:
Nits:
1. Is case insensitive matching really needed? If not can we drop `-i` option?
2. According to `kafka-cruise-control-start.sh` `java` always precedes `KafkaCruiseControlMain`. Can we enforce that in our command?

 -- i.e. would the following work?
```
PIDS=$(ps ax | grep 'java.*KafkaCruiseControlMain' | grep -v grep | awk '{print $1}')
```

Reply:
Thanks for you comment. Good suggestion. I submitted your feedback.